flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js

```diff
@@ -7,7 +7,7 @@ var amadeus = new Amadeus({
 // What were the busiest months for Madrid in 2017?
 amadeus.travel.analytics.airTraffic.busiestPeriod.get({
   cityCode: 'MAD',
-  period: '2017',
```

Comment:
This endpoint should not change, as the API takes as input dates in the past.

Reply:
Resolved the issue.